### PR TITLE
Add travis check for WML indentation correctness.

### PR DESCRIPTION
**Pull request by Pentarctagon**

### CODE REVIEW

File: utils/travis/steps/install.sh [hunk 1]
```diff
@@ -59,14 +59,6 @@ elif [ "$TRAVIS_OS_NAME" = "windows" ]; then
         ./utils/travis/windows-file-hasher.sh "projectfiles/$IMAGE/$CFG/filehashes.sqlite"
     fi
 else
-    # if not doing translations, save a bit of time by not copying them into the docker image
-    # otherwise, if this is the mingw job, the .git directory is needed for running the git archive command
-    if [ "$NLS" == "false" ]; then
-        echo "po/" >> .dockerignore
-    elif [ "$IMAGE" == "mingw" ]; then
-        rm .dockerignore
-    fi
```
Comment (soliton-) [inline utils/travis/steps/install.sh:68]: Was this intended to be part of this PR?

File: utils/travis/docker_run.sh [hunk 2]
```diff
@@ -176,6 +176,10 @@ else
 
     if [ "$VALIDATE" = "true" ]; then
         execute "WML validation" ./utils/travis/schema_validation.sh
+        cd data/tools/
+        make reindent
+        cd ../../
+        execute "WML indentation check" git diff-index --quiet HEAD
```
Comment (soliton-) [inline utils/travis/docker_run.sh:182]: You could make those commands into a function to have them all called by execute:

```suggestion
        checkindent() {
                make -C data/tools reindent &&
                git diff-index --quiet HEAD
        }
        execute "WML indentation check" checkindent
```

File: utils/travis/steps/install.sh (hunk 1 above)
Comment (Pentarctagon) [reply utils/travis/steps/install.sh:68]: Yes - it was a micro-optimization at best to improve the jobs' speed by ~30 seconds, but now that I also need `.git` in another job for this I'd rather not keep adding exceptions.

File: utils/travis/docker_run.sh (hunk 2 above)
Comment (Pentarctagon) [reply utils/travis/docker_run.sh]: Updated.

File: utils/travis/docker_run.sh [hunk 3]
```diff
@@ -174,8 +174,14 @@ else
         fi
     }
 
+    checkindent() {
+            make -C data/tools reindent &&
+            git diff-index --quiet HEAD
```
Comment (stevecotton) [inline utils/travis/docker_run.sh:179]: These lines are indented by 8 spaces, but line 184 (and 183) are indented by 4 spaces. Trivial, but given the subject of this PR it's something that should be fixed.

Comment (Pentarctagon) [reply utils/travis/docker_run.sh]: Updated.

**Pull request merged**